tests/Command/AddUserCommandTest.php

```diff
@@ -33,9 +33,7 @@ protected function setUp()
         exec('stty 2>&1', $output, $exitcode);
         $isSttySupported = 0 === $exitcode;
 
-        $isWindows = '\\' === \DIRECTORY_SEPARATOR;
-
-        if ($isWindows || !$isSttySupported) {
+        if ('Windows' === PHP_OS_FAMILY || !$isSttySupported) {
```

Comment:
you should fully-qualify it, so that it can still be resolved at compile-time when the constant is supported natively.